charts/vsphere-cpi/v1.0.0/values.yaml

```diff
@@ -1,11 +1,31 @@
-vCenter:
+cloudConfig:
   host: ""
-  insecureflag: "1"
-  port: "443"
+  insecureFlag: true
+  port: 443
   datacenters: ""
+  template: |
+    [Global]
+    secret-name = {{ .Values.secret.name | quote }}
+    secret-namespace = {{ .Release.Namespace | quote }}
+    port = {{ .Values.cloudConfig.port | quote }}
+    insecure-flag = {{ .Values.cloudConfig.insecureFlag | quote }}
+
+    [VirtualCenter {{ .Values.cloudConfig.host | quote }}]
+    datacenters = {{ .Values.cloudConfig.datacenters | quote }}
+
+secret:
+  generate: true
+  name: "vsphere-cpi-creds"
   username: ""
   password: ""
-  credentialsSecretName: "vsphere-cpi-creds"
-  generateCredentialsSecret: true
 
-nodeSelector: {}
+cloudControllerManager:
+  image:
+    repository: gcr.io/cloud-provider-vsphere/cpi/release/manager
+    tag: v1.2.1
+
+  nodeSelector: {}
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
```

Comment:
I think this should be at the top of the file

Reply:
Agreed. It was, but probably ended up somewhere else while copy pasting stuff.

Reply:
Agreed. However, I put it at the end to be consistent with our other charts.

https://github.com/rancher/charts/blob/d53552132db09be5fb30db6d1c92d0226d7217b7/charts/rancher-kiali-server/rancher-kiali-server/1.29.000-rc01/values.yaml#L86

https://github.com/rancher/charts/blob/d53552132db09be5fb30db6d1c92d0226d7217b7/charts/rancher-logging/rancher-logging/3.9.000-rc08/values.yaml#L140-L149

Reply:
Ah I wasn't aware that logging and kiali placed it in the bottom of their charts.

Never mind. No need to make this change.